### ISSUE TITLE
Local Android-sdk not found by third party modules

### ISSUE THREAD

**Issue by phyk** [MarcoG3/WhatsDump#25]
Installing the android-sdk via whatsdump works fine, however when continuing the process the third party android python module runs into the problem of not finding the adb. Adding the android-sdk to the path environment fixed this issue. Could that simply be included into the python code?

**Comment by MarcoG3** [comment]
Can you give additional informations about this ADB issue? Are you running the tool on Windows?

Do you have the following file in your filesystem `./WhatsDump/android-sdk/platform-tools/adb` or something similar?

**Comment by phyk** [comment]
Yes I am running it on windows 10. However, this issue also arose on a test environment on a Mac.

No this file is not present in the platform-tools folder generated by whatsdump

**Comment by phyk** [comment]
Sorry, i rechecked, there is a file adb.exe

**Comment by MarcoG3** [comment]
Should be fixed in 482ad5e, let me know if the same issue occurs again!

**Comment by phyk** [comment]
I will check this tomorrow, my whatsapp is out of retries for registering the phone
